suite-native/module-send/src/components/TokenOfNetworkBottomSheet.tsx

```diff
@@ -0,0 +1,152 @@
+}: {
+    accountKey: AccountKey;
+    tokenContract?: TokenAddress;
+}) => {
+    const { applyStyle } = useNativeStyles();
+    const tokenSymbol = useSelector((state: TokensRootState) =>
+        selectAccountTokenSymbol(state, accountKey, tokenContract),
+    );
+    const networkSymbol = useSelector((state: AccountsRootState) =>
+        selectAccountNetworkSymbol(state, accountKey),
+    );
+
+    if (!tokenContract || !networkSymbol) return null;
+
+    const networkName = getNetwork(networkSymbol).name;
+
+    return (
+        <VStack spacing="sp24">
+            <Box style={applyStyle(iconWrapperStyle)}>
+                <CryptoIcon symbol={tokenContract} size={80} />
+                <Box style={applyStyle(networkIconWrapperStyle)}>
+                    <CryptoIcon symbol={networkSymbol} size={32} />
+                </Box>
+            </Box>
+            <Text variant="titleSmall">
+                <Translation
+                    id="moduleSend.outputs.tokenOfNetworkSheet.title"
+                    values={{ tokenSymbol, networkName }}
+                />
+            </Text>
+            <Paragraph
+                header={
+                    <Translation id="moduleSend.outputs.tokenOfNetworkSheet.body.self.subtitle" />
+                }
+                body={
+                    <Translation
+                        id="moduleSend.outputs.tokenOfNetworkSheet.body.self.text"
+                        values={{ networkName }}
+                    />
+                }
+            />
+            <Paragraph
+                header={
+                    <Translation id="moduleSend.outputs.tokenOfNetworkSheet.body.outside.subtitle" />
+                }
+                body={
+                    <Translation
+                        id="moduleSend.outputs.tokenOfNetworkSheet.body.outside.text"
+                        values={{ networkName }}
+                    />
+                }
+            />
+            <AlertBox
+                title={
+                    <Text variant="callout" color="textDefault">
+                        <Translation id="moduleSend.outputs.tokenOfNetworkSheet.warning" />
+                    </Text>
+                }
+                variant="warning"
+                borderRadius="r12"
+            />
+        </VStack>
+    );
+};
+
+export const useTokenOfNetworkAlert = () => {
+    const wasAlertShown = useRef(false);
+    const { showAlert } = useAlert();
+    const {
+        params: { tokenContract, accountKey },
+    } = useRoute<RouteProp<SendStackParamList, SendStackRoutes.SendOutputs>>();
+
+    const tokenSymbol = useSelector((state: TokensRootState) =>
+        selectAccountTokenSymbol(state, accountKey, tokenContract),
+    );
+    const networkSymbol = useSelector((state: AccountsRootState) =>
+        selectAccountNetworkSymbol(state, accountKey),
+    );
+
+    const { watch } = useFormContext();
+
+    const addressValue = watch(getOutputFieldName(0, 'address'));
```

Comment:
There should be an index passed from the parent component. This would brake in the case of multi-recipient sendform.